main/entry.c

```diff
@@ -1190,7 +1190,7 @@ static bool isTagWritable(const tagEntryInfo *const tag)
 			return false;
 
 		/* TODO: optimization
-		   A Bitmasks represeting all enabled roles can be calculated at the
+		   A Bitmasks represetting all enabled roles can be calculated at the
```

Comment:
What I would like to write was "representing".

Reply:
Updated.